src/peasy/PeasyCam.java

```diff
@@ -498,11 +498,23 @@ public void reset(final long animationTimeInMillis) {
 				animationTimeInMillis);
 	}
 
-	public void pan(final double dx, final double dy) {
-		center = center.add(rotation.applyTo(new Vector3D(dx, dy, 0)));
+	public void move(final double dx, final double dy, final double dz) {
+		center = center.add(rotation.applyTo(new Vector3D(dx, dy, dz)));
 		feed();
 	}
 
+	public void pan(final double dx, final double dy) {
+		move(dx, dy, 0);
+	}
+
+	public void forward(final double distance) {
+		move(0, 0, -distance);
+	}
+
+	public void backward(final double distance) {
```

Comment:
Naybe rename forward/backward to be pushIn, pushOut? That's what the relevant camera moves are called IRL. :)

Reply:
Yes good idea !

I thought about changing the methods name and was trying to come up with single word for "depth -or- front/back travelling" to which we could feed negative or positive value. But pushIn/out is perfect !

We could also get rid of `pushIn()` and `pushOut()` entirely and use `move()` only ?

I'll rename them for now, don't hesitate to tell me what you think.